docs/aerodromes/classc/Brisbane.md

```diff
@@ -29,6 +29,74 @@ There are three SMC positions at Brisbane which are responsible for various part
   <figcaption>YBBN Maneuvering Area Responsibility</figcaption>
 </figure>
 
+## Helicopter Operations
+Brisbane has one helipad located within the general aviation apron, most rescue helicopters are based within the general aviation maintenance area and will depart from the **H2 taxiway** to save time when operating.
+
+#### Waypoints
+VFR and IFR helicopters are normally processed via two visual waypoints north and south of the field, when in VMC conditions. IFR helicopters in IMC conditions may conform to fixed wing ops and be processed via the BN (RADAR) SID from an appropriate runway, unless a visual departure is acceptable. These waypoints are used for **departures** and **arrivals**, depending on weather and traffic conditions.
+
+
+| Direction               | Waypoint       | Name        |
+| ------------------ | -------------- | ---------------- |
+| North      | BLHS   | Bald Hills          |
+| South      | MBHM   | Manly Boat Harbour          |
+| East      | MBHM   | Manly Boat Harbour          |
+| West      | BLHS   | Bald Hills          |
+
+#### Departures
+
+Refer to [Maneuvering Area Responsibility](#maneuvering-area-responsibility) to determine which SMC or ADC position (when non-standard positions are online) is responsible for managing helicopter arrivals and departures.
+
+!!! example
+    **RSCU533:** "Brisbane Delivery, helicopter RSCU533, medivac for YXHE, request clearance"  
+    **BN ACD:** "RSCU533, Brisbane Delivery, cleared to YXHE via BLHS, climb 2500, squawk 1256, departure frequency 124.7"  
+    **RSCU533:** "Cleared to YXHE via BLHS, climb 2500, squawk 1256, departures 124.7, RSCU533"
+
+Taxiway H2 is inside the maneuvering area and therefore owned by **BN SMC-N**, treated like a runway requiring a takeoff and landing clearance by BN ADC. Helicopters departing on other surfaces outside the maneuvering areas should be instructed to *"report airborne"* so the controllers knows when they are active in their airspace. 
+
+!!! example
+    **RSCU533:** "Brisbane Tower, helicopter RSCU533, H2 ready"  
+    **BN ADC:** "RSCU533, Brisbane Tower, H2 cleared for takeoff"  
+    **RSCU533:** "H2 cleared for takeoff, RSCU533"
+ 
+    **BN ADC:** "RSCU533, contact departures"  
+    **RSCU533:** "Departures, RSCU533"
+
+#### Arrivals
+
+IFR helicopters operating within IMC conditions may conform to fixed wing ops and be processed via an instrument approach for a appropriate duty runway. Standard fixed wing phraseology shall be used during these conditions.
+
+!!! example
+    **RSCU533:** "Brisbane Tower, helicopter RSCU533"  
+    **BN ADC:** "RSCU533, Brisbane Tower, cleared visual approach H2"  
+    **RSCU533:** "Cleared visual approach H2, RSCU533"
+
+During times of peak fixed wing traffic, instruct helicopters to orbit in present position or overhead the VFR waypoint until advised, and maintain own separation on approach to the helipad.
```

Comment:
Are the heli waypoints separated from fixed wing ops? If so, we might suggest that as a course of action exclusively, rather than present position.

Reply:
The heli waypoints are separated from fixed wing ops, the actual suburb / waypoint of Bald Hills is 6nm north of 01L/19R. I only added the present position call in case the helis are pasted Bald Hills are over some of the closer suburbs.  But from what i've heard IRL, Bald Hills doesn't necessarily act as a clearance limit, just an inbound reporting point. However I may be wrong and it could be important that we mention it similar to how Sydney's coded arrivals work with holding at the limit.

Reply:
Ah right, that's good then. There's also a likelihood that these aircraft will enter the CTR directly from class G. I suspect we'll end up going down the line of TMA clearing them to say A010V and issuing a clearance limit of the waypoint, then contacting TWR for visual approach clearance (who will instruct them to remain clear of any fixed wing traffic if necessary). If the aircraft is going Class G->CTR, they will likely be told to remain OCTA until that takes place. Hold tight for now and we'll see whether we want to redo the BN CTR jurisdiction